### DISCUSSION TITLE
application (CA irrigation detection)

### DISCUSSION THREAD

**Discussion by taeil** [Berkeley-Data/irrigation_detection#4] [General]
- [surya, ken] demonstrate better detection
section **6.2 Unsupervised Model Using California Data** is pretty weak  
- irrigation 30 (pre-prev team)

**Comment by suryagutta** [comment]
- For the model validation  (if we still have to go with CA data), any thoughts on the effective way to help the quality of the publishable paper? Do we have to label the CA data manually for validation? We couldn't find labeled data for CA agriculture.

- We are using SimCLR (or another existing framework for better performance) using satellite imagery in the current project. We prove that SimCLR performs well with Satellite imagery also. Is this OK for the paper to be publishable, or do we have to come-up with any innovative way utilizing the SimCLR framework?   

- Performance comparison:  In the attached image, we are comparing 64 images. But, in reality, we might be having 64 * 5 (assume 5 augmentations) = 320 images. I can argue that I can apply the same augmentations and dynamically label those new images based on the main image. Now, I will have 320 labeled images which can be used by the supervised model. Here, I am not talking about whether it's a positive or negative image pair as SymCLR uses. But, it's a straight forward image generation and labeling (no manual labeling). I think the supervised performance might increase and won't be worse.
![image](https://user-images.githubusercontent.com/15736461/106673890-36c74d00-6567-11eb-83a8-b2be08b3d989.png)

- What's the website's priority, since it has 20% of the total grade,  considering the existing website and publishable paper?
